OpenRA.Mods.Common/Widgets/Logic/MapChooserLogic.cs

```diff
@@ -33,8 +33,18 @@ internal MapChooserLogic(Widget widget, string initialMap, Action onExit, Action
 		{
 			selectedUid = WidgetUtils.ChooseInitialMap(initialMap);
 
-			var approving = new Action(() => { Ui.CloseWindow(); onSelect(selectedUid); });
-			var canceling = new Action(() => { Ui.CloseWindow(); onExit(); });
+			var approving = new Action(() =>
+			{
+				Ui.CloseWindow();
+				onSelect(selectedUid);
+				Ui.Root.Get<TextFieldWidget>("CHAT_TEXTFIELD").TakeKeyboardFocus();
```

Comment:
This is for when the user closes map chooser. So that the chat text field has focus when returned to lobby window.